FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+        /// StartFind supports, HTML, PDF, and TXT document queries. In general this api is designed for text based find sessions.
+        //// If you start a find session programmatically on another file format that doesnt have text fields, the find session will try to execute but will fail to find any matches. (It will silently fail)
+        /// Note: The asynchronous action completes when the UI has been displayed with the find term in the UI bar, and the matches have populated on the counter on the find bar. 
+        /// There may be a slight latency between the UI display and the matches populating in the counter. 
+        /// The MatchCountChanged and ActiveMatchIndexChanged events are only raised after StartFindAsync has completed, otherwise they will have their default values (-1 for both).
+        /// When initiating a find session while another session is in progress, the behavior of 
+        /// the active match index depends on the direction set for the find operation (forward or backward).
+        /// However, calling StartFind again during an ongoing find operation does not resume from the point 
+        /// of the current active match. For example, given the text "1 1 A 1 1" and initiating a find session for "A",
+        /// then starting another find session for "1", it will start searching from the beginning of the document, 
+        /// regardless of the previous active match. This behavior indicates that changing the find query initiates a 
+        /// completely new find session, rather than continuing from the previous match index. This distinction is essential 
+        /// to understand when utilizing the StartFind method for navigating through text matches.
+        Windows.Foundation.IAsyncAction StartFindAsync(CoreWebView2FindConfiguration configuration);
+
+
+        /// Navigates to the next match in the document.
+        /// If there are no matches to find, FindNext will wrap around to the first match if the search direction is forward, 
+        /// or to the last match if the search direction is backward.
+        /// If called when there is no find session active, FindNext will silently fail.
+        void FindNext();
+        
+        /// Navigates to the previous match in the document.
+        /// If there are no matches to find, FindPrevious will wrap around to the last match if the search direction is forward, 
+        /// or to the first match if the search direction is backward.
+        /// If called when there is no find session active, FindPrevious will silently fail.
+        void FindPrevious();
+
+
+        /// Stops the current 'Find' operation and hides the Find bar.
+        /// If called with no Find session active, it will silently do nothing.
+        void StopFind();
+
+        /// Retrieves the index of the currently active match in the find session. Returns the index of the currently active match, or -1 if there is no active match.
```

Comment:
Does this repo support IReference for "may not exist" primitives, instead of a sentinel value?

Reply:
Leave as is. Discussion: We can support nullable / ireference with a bunch of additional work but don't find it to be ROI worthwhile